src/components/Header/index.js

```diff
@@ -16,6 +16,15 @@ const Header = ({ className, isDark, isLight, authStore, appStore }) => {
     </svg>
   );
 
+  const SettingsIcon = ({ ...props }) => (
+    <svg xmlns='http://www.w3.org/2000/svg' width='25' height='25' className={style.settingsIcon} {...props}>
+      <g className={style.settingsIconFill} fillRule='evenodd'>
+        <path d='M12.499.064C5.64.064.065 5.648.065 12.504c0 6.858 5.575 12.433 12.434 12.433 6.857 0 12.435-5.575 12.435-12.432S19.356.064 12.499.064zm0 23.591c-6.151 0-11.156-5-11.156-11.151 0-6.159 5.006-11.159 11.156-11.159 6.147 0 11.154 5 11.154 11.159 0 6.15-5.005 11.151-11.154 11.151z' />
+        <path d='M13.14 6.844V4.783a.638.638 0 0 0-.641-.64.639.639 0 0 0-.643.64v2.061a5.7 5.7 0 0 0-5.05 5.66 5.697 5.697 0 0 0 5.693 5.692 5.7 5.7 0 0 0 5.694-5.692 5.701 5.701 0 0 0-5.054-5.66zm-.641 10.071a4.42 4.42 0 0 1-4.416-4.411 4.42 4.42 0 0 1 3.774-4.366v4.366c0 .35.288.633.643.633.352 0 .64-.282.64-.633V8.137a4.421 4.421 0 0 1 3.778 4.367 4.424 4.424 0 0 1-4.42 4.411z' />
+      </g>
+    </svg>
+  );
+
```

Comment:
it could be better if this svg variable would be in separate file :)
but since it is a very small app I can turn a blind eye to it ;)